packages/lesswrong/components/ultraFeed/UltraFeedCommentsDialog.tsx

```diff
@@ -160,6 +215,37 @@ const UltraFeedCommentsDialog = ({
   const comments = isPost ? postComments : threadComments;
   const totalCount = isPost ? postCommentsTotalCount : threadCommentsTotalCount;
 
+  // Handle browser back button / swipe back navigation
+  useEffect(() => {
```

Comment:
Is this identical to the other place you're doing it?  If so, extract to shared hook in `components/hooks`